go/cli/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/cli/commands"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	var err error
+	cmd := &cobra.Command{
+		Use:   "kfn",
+		Short: "a CLI tool to edit your own KRM functions with ease",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			h, err := cmd.Flags().GetBool("help")
```

Comment:
Nit: I don't think you need to do this?  I think the behaviour if you just comment out the RunE does basically this

Reply:
thanks. cleanedup.